index.android.js

```diff
@@ -22,6 +23,12 @@ var MusicControl = {
     NativeMusicControl.enableBackgroundMode(enable)
   },
   setNowPlaying: function(info){
+    //Check if we have an android asset id from react style image "require('./image.png')"
+    if (info.artwork && !isNaN(info.artwork)) {
+      info.artwork = resolveAssetSource(info.artwork).uri;
```

Comment:
We have to resolve the asset on the JS side as the resolveAssetSource function does not have a java equivalent. If I understand you we should remove the ".uri" and then check on java side if it is a map and extract uri that way?